shell/app/modules/project/pages/test-manage/case/header/import-export-record.tsx

```diff
@@ -38,13 +38,14 @@ const ImportExportRecord = ({
   const [isFinished, setIsFinished] = useState(false);
   const loginUser = userStore.useStore((s) => s.loginUser);
   const [list, setList] = useState([] as TEST_CASE.ImportExportRecordItem[]);
+  const [counter, setCounter] = useState({} as TEST_CASE.ImportExportCounter);
```

Comment:
dont set `{}` as init value
```
const [counter, setCounter] = useState<TEST_CASE.ImportExportCounter | null>(null);
```

then u just need to check whether `counter` exist, dont need to check `Object.keys(counter).length`

Reply:
when counter is empty, api will return 'counter: {}',  like this
![image](https://user-images.githubusercontent.com/30014895/127444299-1e644ef8-a6ba-4836-a076-e2fa38c33ecf.png)